Tofu/AccountCell.swift

```diff
@@ -28,39 +28,47 @@ private func placeholderImageWithText(_ text: String) -> UIImage {
 }
 
 private func imageForAccount(_ account: Account) -> UIImage {
-    switch account.issuer {
+    let identifier = account.issuer != nil ? account.issuer : account.description
+    
+    switch identifier {
     case .some("Adobe ID"): return UIImage(named: "Adobe")!
     case .some("Allegro"): return UIImage(named: "Allegro")!
     case .some("Amazon"): return UIImage(named: "Amazon")!
-    case .some("AWS"): return UIImage(named: "AWS")!
+    case .some("AWS"): fallthrough
+    case .some("Amazon Web Services"): return UIImage(named: "AWS")!
```

Comment:
What do you think about combining these cases instead of using `fallthrough`?

Something like this:

```swift
case .some("AWS"), .some("Amazon Web Services"): return UIImage(named: "AWS")!
```